app/models/energy/construction.py

```diff
@@ -437,178 +493,153 @@ class EnergyWindowMaterialBlind(BaseModel):
         ' plastic slats. The default value is 0.'
     )
 
-    front_infrared_hemispherical_emissivity: float = Schema(
+    front_infrared_hemispherical_emissivity: float=Schema(
         0.9,
         ge=0,
         lt=1,
         description='Front side hemispherical emissivity of the slat. Default is 0.9 for'
         ' most materials. The default value is 0.9.'
     )
 
-    back_infrared_hemispherical_emissivity: float = Schema(
+    back_infrared_hemispherical_emissivity: float=Schema(
```

Comment:
Let's just call this `back_emissivity` for consistency with all of the other material names.